nemoguardrails/colang/v2_x/library/llm.co

```diff
@@ -95,6 +95,9 @@ flow continuation on unhandled user utterance
 
   log 'start generating user intent and bot intent/action...'
   $action = 'user said "{$event.final_transcript}"'
+
+  # NOTE: I am not sure if this is needed here.
+  await RetrieveRelevantChunksAction()
```

Comment:
Yes, this is needed here. The relevant chunks need to be computed before the generation happen in `GenerateUserIntentAndBotAction`.